server/routes/index.js

```diff
@@ -474,38 +475,39 @@ module.exports = function Index ({ authenticationMiddleware }) {
     res.render('match-manual', templateValues)
   })
 
-  router.post('/:courtCode/case/:caseId/match/defendant/:defendantId/manual', body('crn').trim().escape(), defaults, async (req, res) => {
-    const { params: { courtCode, caseId, defendantId }, body: { crn }, session } = req
+  router.post('/:courtCode/case/:caseId/hearing/:hearingId/match/defendant/:defendantId/manual', body('crn').trim().escape(), defaults, async (req, res) => {
+    const { params: { courtCode, caseId, defendantId, hearingId }, body: { crn }, session } = req
     let redirectUrl = '/'
```

Comment:
Change this to `let redirectUrl=`/${courtCode}/case/${caseId}/hearing/${hearingId}/match/defendant/${defendantId}/manual`